sdks/python/apache_beam/examples/streaming_wordcount.py

```diff
@@ -42,22 +42,31 @@ def split_fn(lines):
 def run(argv=None):
   """Build and run the pipeline."""
   parser = argparse.ArgumentParser()
-  parser.add_argument(
-      '--input_topic', required=True,
-      help=('Input PubSub topic of the form '
-            '"projects/<PROJECT>/topics/<TOPIC>".'))
   parser.add_argument(
       '--output_topic', required=True,
       help=('Output PubSub topic of the form '
             '"projects/<PROJECT>/topic/<TOPIC>".'))
+  group = parser.add_mutually_exclusive_group(required=True)
+  group.add_argument(
+      '--input_topic',
+      help=('Input PubSub topic of the form '
+            '"projects/<PROJECT>/topics/<TOPIC>".'))
+  group.add_argument(
+      '--input_subscription',
+      help=('Input PubSub subscription of the form '
+            '"projects/<PROJECT>/subscriptions/<SUBSCRIPTION>."'))
   known_args, pipeline_args = parser.parse_known_args(argv)
   options = PipelineOptions(pipeline_args)
   options.view_as(StandardOptions).streaming = True
 
   with beam.Pipeline(options=options) as p:
 
     # Read from PubSub into a PCollection.
-    lines = p | beam.io.ReadStringsFromPubSub(known_args.input_topic)
+    if known_args.input_subscription:
+      lines = p | beam.io.ReadStringsFromPubSub(
+          subscription=known_args.input_subscription)
```

Comment:
@aaltay The line wrap looks a bit ugly. If there is a better way to do, I'd love to have a quick change.

Reply:
It is fine as long as it is lint compliant.